app/views/foreman/smart_proxies/_content_tab.html.erb

```diff
@@ -8,9 +8,11 @@
       <th translate>Last Published</th>
       <th translate>Hosts</th>
       <th translate>Products</th>
+      <th translate>Apt repos</th>
       <th translate>Yum repos</th>
       <th translate>Container Image repos</th>
-      <th translate>Packages</th>
+      <th translate>Debian Packages</th>
+      <th translate>Redhat Packages</th>
```

Comment:
'Red Hat Packages' :)

Reply:
I'd actually prefer 'RPM Packages'   but good job @jturel for keeping brand in mind ;)

Reply:
yeah good call on 'RPM Packages'!

Reply:
RPM Packages it is... ;)
Also changed Debian (the distribution) to Deb (the file format).